### ISSUE TITLE
Link to one topo from another

### ISSUE THREAD

**Issue by oliverstory** [theCrag/website#1981]
This is a related feature request to #1693 and also I think something you've looked into in the past with #573

I have a large photo showing a multipitch cliff at Bungonia https://www.thecrag.com/11771389/topo?edit=711052251: topo #10177 

The top part of the cliff the routes come close together and so would be good to show a link to a topo of the top wall (see Topo #10176)  in the same way you'd show a link to an area.

There may be other solutions making this redundant e.g. allowing more zoom levels so the user can really zoom in on part of a topo (in this example the photos are the same but cropped differently).

**Comment by brendanheywood** [comment]
Yup +1. The root problem here is that topos are currently a 'node' like a route and area. We need to turn topos into a node so we can also set the order of them between routes and areas manually as well, see #1190. Once that is done linking from a topo to another topo should be pretty straight forward.

**Comment by scd** [comment]
FYI, I don't think we need to make the topo a node from a database perspective. A while ago I changed the topo routes database field into a generic object - actually it is still called routes in the database, but it can accept any object id.

I think this means it is just a UI issue, and data templating issue.

**Comment by brendanheywood** [comment]
s/node/object/ ...  :)
